### PR TITLE
feat added redux

### PR DESCRIPTION
**Pull request by deepak29-git**
Added redux toolkit

### CODE REVIEW

File: src/all-api/follow-unfollow-api.js [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
+const unfollowUserApi = async (followUserId, dispatch) => {
   try {
     const { data } = await axios({
       method: "POST",
       url: `/api/users/unfollow/${followUserId}`,
       data: {},
       headers: { authorization: getToken() },
     });
-    userDispatch({ type: "UNFOLLOW_USER", payload: data.user });
+    // userDispatch({ type: "UNFOLLOW_USER", payload: data.user });
```
Comment (nerdyspook) [inline src/all-api/follow-unfollow-api.js:26]: You can remove this comment from here.

**Review by nerdyspook — APPROVED** [review]
Liked the use of dispatch function.

File: src/redux/features/posts/postSlice.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    commentId:"",
+    status:"idel",
+    
+}
+
+
+// export const loadPosts=createAsyncThunk("posts/loadPosts",async()=>{
+//     const { data } = await axios.get("/api/posts");
+//     return data.posts
+// })
```
Comment (prathmeshjagtap) [inline src/redux/features/posts/postSlice.js:20]: you can remove commented code if not required

**Review by prathmeshjagtap — COMMENTED** [review]
Code Is clean , UI is also good , good to merge

**Review by prathmeshjagtap — APPROVED** [review]

File: src/all-api/follow-unfollow-api.js (hunk 1 above)
Comment (deepak29-git) [reply src/all-api/follow-unfollow-api.js:26]: ok

File: src/redux/features/posts/postSlice.js (hunk 2 above)
Comment (deepak29-git) [reply src/redux/features/posts/postSlice.js:20]: ok

**Review by nishasen — APPROVED** [review]
Looks good and clean code

**Pull request merged**